build.sbt

```diff
@@ -43,7 +43,9 @@ lazy val microservice = (project in file("."))
 val compileDependencies = Seq(
   "com.github.pureconfig"   %% "pureconfig"                   % "0.12.3",
   "uk.gov.hmrc"             %% "bootstrap-backend-play-26"    % "2.24.0",
-  "uk.gov.hmrc"             %% "simple-reactivemongo"         % "7.26.0-play-26"
+  "uk.gov.hmrc"             %% "simple-reactivemongo"         % "7.30.0-play-26",
+  "org.mongodb.scala"       %% "mongo-scala-driver"           % "2.9.0",
+  "org.mongodb"             %  "mongo-java-driver"            % "3.5.0"
```

Comment:
Why are there both java and scala drivers?

Reply:
The new migration code uses 'com.mongodb.client.MongoDatabase' from the java driver that I think mongock used to bring in as a dependency. I had a quick go at swapping to the scala driver in the code but due to slight differences in implementation of the drivers it was a non-trivial change so took more pragmatic approach. Maybe a tech debt job to migrate to scala driver in future :wink:

Reply:
The new migration code uses 'com.mongodb.client.MongoDatabase' from the java driver that I think mongock used to bring in as a dependency. I had a quick go at swapping to the scala driver in the code but due to slight differences in implementation of the drivers it was a non-trivial change so took more pragmatic approach :)